shells/pipes-shell/web/deploy/source/index.html

```diff
@@ -15,6 +15,8 @@
 
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+  <!-- No complains about missing favicon.ico from https requests. -->
+  <link rel="icon" href="data:,">
```

Comment:
Thank you! This had been bugging me too 😛

Reply:
Np!